python/cudf/cudf/core/column/lists.py

```diff
@@ -21,6 +22,36 @@ def __init__(
             children=children,
         )
 
+    def __sizeof__(self):
+        n = 0
+        if self.nullable:
+            n += cudf._lib.null_mask.bitmask_allocation_size_bytes(
+                self.size - self.offset
```

Comment:
Isn't `self.size` applied on top of the offset? so if offset is 5 and the size is 5, the size of the new column is still 5

Reply:
Yes, this was incorrect - `size` is inclusive of `offset` value